src/test/java/api/loans/anonymization/AnonymizeLoansAfterXIntervalTests.java

```diff
@@ -432,4 +434,19 @@ public void testClosedLoanWithOpenFeesAndFinesNotAnonymizedIntervalPassed()
     assertThat(loansStorageClient.getById(loanID)
         .getJson(), not(isAnonymized()));
   }
+
+  @Test
+  public void testNoConfigurationExceptionIsHandled()
+    throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+
+    IndividualResource loanResource = loansFixture.checkOutByBarcode(new CheckOutByBarcodeRequestBuilder().forItem(item1)
+      .to(user)
+      .at(servicePoint.getId()));
+    UUID loanID = loanResource.getId();
+
+    Response response = anonymizeLoansInTenant();
+    assertThat(response.getStatusCode(), is(422));
```

Comment:
Can the response itself also be checked?

Reply:
It could be checked. What do you suggest to check? The presence of `error` there?